docker/Makefile

```diff
@@ -1,5 +1,14 @@
+# Variables can be overridden by setting environment variables
+
+FACTORIO_PATH ?= ~/.factorio/
+SECURITY_PATH ?= $(FACTORIO_PATH)/security
+SAVES_PATH ?= $(FACTORIO_PATH)/saves
+MODS_PATH ?= $(FACTORIO_PATH)/mods
+ADMIN_PASS ?= jqkSnQS4rA
```

Comment:
Can you please either remove this var completly (the container will autogenerate a new somewhat save password) or figure something out to prevent people from using this default? I've the really bad feeling that there might be some instances with this password otherwise...
Mostly because people are stupid and don't care to change it. We should either force them to change it or allow them not to care (by autogenerating one).